Framework/PythonInterface/mantid/api/src/Exports/SpectrumDefinition.cpp

```diff
@@ -0,0 +1,36 @@
+#include "MantidTypes/SpectrumDefinition.h"
+
+#include <boost/python/class.hpp>
+#include <boost/python/tuple.hpp>
+
+using Mantid::SpectrumDefinition;
+using namespace boost::python;
+
+// Helper function to convert a std::Pair to a Python tuple
+boost::python::tuple convertPairToTuple(const SpectrumDefinition &self,
+                                        const size_t index) {
+  const std::pair<size_t, size_t> pair = self.operator[](index);
+  boost::python::tuple ret_val = make_tuple(pair.first, pair.second);
```

Comment:
You can avoid the temporary and just return directly from `make_tuple`

Reply:
Ah yes of course!